frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -414,6 +418,7 @@ export default class NewForm extends Vue {
   @Prop() searchResultTeamsGroups!: types.ListItem[];
   @Prop({ default: false }) contestForTeams!: boolean;
   @Prop({ default: null }) problems!: types.ProblemsetProblemWithVersions[];
+  @Prop() hasVisitedSection!: boolean;
```

Comment:
Acá para que sólo nos muestre el modal de introJS sólo en la vista de crear concurso hay que configurar un valor por defecto `true` a la propiedad:
```suggestion
  @Prop({ default: true }) hasVisitedSection!: boolean;
```